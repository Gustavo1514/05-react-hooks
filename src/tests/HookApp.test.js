const { shallow } = require('enzyme')
const { default: HookApp } = require('../HooksApp')

describe('Pruebas sobre HookApp', () => {
  test('debe renderizar corectamente', () => {
    let wrapper = shallow(<HookApp />)
    expect(wrapper).toMatchSnapshot()
  })
})
