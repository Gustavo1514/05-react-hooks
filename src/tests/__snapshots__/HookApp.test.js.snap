// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas sobre HookApp debe renderizar corectamente 1`] = `
<div>
  <h1>
    Hola
  </h1>
</div>
`;
